src/toil/toilState.py

```diff
@@ -40,51 +42,56 @@ def __init__(self, jobStore, rootJob, jobCache=None):
         the JobStore, and is not preserved.
 
         :param toil.jobStores.abstractJobStore.AbstractJobStore jobStore
+
         :param toil.job.JobDescription rootJob
+
+        :param Dict[str, JobDescription] jobCache
```

Comment:
Rather than adding param lines that duplicate the types (and can be different) with no explanation of the meaning, I would drop the types form these and explain what the parameter is for. If you can't tell what the parameter is for and you moved the type to the arguments, I guess you can just drop the line.


```
:param jobStore: The job store to use.
:param rootJob: The description for the root job of the workflow being run.
:param jobCache: A dict to cache downloaded job descriptions in, keyed by ID.
```